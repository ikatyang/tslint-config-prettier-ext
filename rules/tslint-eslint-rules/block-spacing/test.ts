function foo() {return true;}
if (foo) { bar = 0;}
