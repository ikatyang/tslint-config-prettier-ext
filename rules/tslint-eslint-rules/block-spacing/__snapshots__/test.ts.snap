// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
function foo() {return true;}
if (foo) { bar = 0;}

======
function foo() {
  return true;
}
if (foo) {
  bar = 0;
}

>>>>>> after
"
`;

exports[`should fail before formatting 1`] = `
"ERROR: (block-spacing) /rules/tslint-eslint-rules/block-spacing/test.ts[1, 16]: Requires a space
ERROR: (block-spacing) /rules/tslint-eslint-rules/block-spacing/test.ts[2, 10]: Requires a space
"
`;
